### ISSUE TITLE
free Tomorrow io api

### ISSUE THREAD

**Issue by fearmusa** [breezy-weather/breezy-weather#469]
### Link to publicly available documentation of the API

https://docs.tomorrow.io/reference/welcome

### Available and not available data

Örnek api: 
https://api.tomorrow.io/v4/timelines?location=xlocation,ylocation&fields=temperature&timesteps=1h&units=metric&apikey=xxxxxxxxxxxx

Tomorrow.io’s first satellite and upcoming constellation is not only revolutionizing weather forecasting but bridging a radar coverage gap for five billion people—all while transforming data inputs for our cutting-edge weather intelligence technology.

### Are you going to develop yourself the new source for the app?

no I am a beginner

### Give a bit more details or context about this source

I am using the application of this website. so far I have obtained very accurate results. I think this information is obtained with a special satellite. Minute by minute precipitation and daily forecasts are very useful. reverse location information is available. I have been testing for a while. api key can be created by creating a free membership. weather forecast expressions such as accuweather are available in the application. for example "cloudy around noon and then mainly clear". I wish I could add this api to the application as the main source.

### Acknowledgements

- [X] This source has a free-tier available (no mandatory credit card info), and if it a main weather source, it includes free hourly forecast
- [X] If this is a weather source request, I checked that endpoints are by coordinates, or that there is a reverse geocoding endpoint.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written the name of the source as the title of this issue.
- [X] I have updated the app to **[version 4.6.5 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)** and this source is not part of it.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
What's the rate limit for free users?

**Comment by fearmusa** [comment]
> What's the rate limit for free users?

I couldn't find any value regarding this part. So I couldn't see it. There could possibly be a limit. Friends who are knowledgeable about the subject can review the documents. That's why I wanted to share.

**Comment by fearmusa** [comment]
> What's the rate limit for free users?

i think i found an article about this on a website. i'm not sure, but the following values are more than enough for a standard user.

source: https://pypi.org/project/tomorrow-io/

"Note that the free plan limits you to 500 requests/day and 25 requests/hour."

**Comment by papjul** [comment]
OK, so this source can be implemented but it will require users to register their own key, so it has a lower priority than sources where we can include a free API key in the app.

**Comment by fearmusa** [comment]
> OK, so this source can be implemented but it will require users to register their own key, so it has a lower priority than sources where we can include a free API key in the app.

yes you are right. the site's own application does not work very healthy, that is, it has problems. your application works much more stable. i want to add it to your application as a private api and use it. i hope it can be added as a feature as soon as possible. big companies around the world use this system.

**Comment by fearmusa** [comment]
This API may not be accurate regarding temperature. I agree with this. But it is good for minute by minute rainfall.

It's worked out well so far in terms of rain chances. Since there are not many storms in my country, I am not sure about this. The data provided by the API is available in the images.

![Screenshot_20230928-121204_Weather](https://github.com/breezy-weather/breezy-weather/assets/127376654/9b034576-0426-4f35-8588-832bc6bfe120)
![Screenshot_20230928-121116_Weather](https://github.com/breezy-weather/breezy-weather/assets/127376654/cb1b4e32-0c77-43c0-ba63-47ba2f990c81)

**Comment by fearmusa** [comment]
![Screenshot_20230928-121116_Weather](https://github.com/breezy-weather/breezy-weather/assets/127376654/bc0a67d1-5022-4fae-b4a1-96a6d0b5da87)
![Screenshot_20230928-121204_Weather](https://github.com/breezy-weather/breezy-weather/assets/127376654/6a7a9737-cc56-475a-ba1d-e8c225f3dee0)
